custom/icds/rules/custom_actions.py

```diff
@@ -36,15 +37,17 @@ def _update_existing_tech_issue_delegate(tech_issue_delegate, owner_id):
     else:
         change_in_level = '1'
 
-    return update_case(
-        tech_issue_delegate.domain,
-        tech_issue_delegate.case_id,
-        case_properties={'change_in_level': change_in_level},
-        close=False,
-        xmlns=AUTO_UPDATE_XMLNS,
-        device_id=__name__ + "._update_existing_tech_issue_delegate",
-        owner_id=owner_id,
-    )
+    kwargs = {
+        "case_properties": {'change_in_level': change_in_level},
+        "close": False,
+        "xmlns": AUTO_UPDATE_XMLNS,
+        "device_id": __name__ + "._update_existing_tech_issue_delegate",
+    }
+    if toggles.ICDS_AUTO_ESCALATION_QA.enabled(tech_issue_delegate.domain):
+        kwargs.update({
+            "owner_id": owner_id,
```

Comment:
There is no owner_id passed if the flag is disabled?

Reply:
Yeah, using owner id is the main part of the change.

Reply:
Rather, there are two main parts of the change - the one below that updates the logic to find the existing delegate, and then this part that updates the delegate's owner id as part of reusing/escalating the delegate.